cms/envs/production.py

```diff
@@ -540,3 +540,10 @@ def get_env_setting(setting):
 ########################## Derive Any Derived Settings  #######################
 
 derive_settings(__name__)
+
+############## Settings for Completion API #########################
+
+# Once a user has watched this percentage of a video, mark it as complete:
+# (0.0 = 0%, 1.0 = 100%)
+COMPLETION_VIDEO_COMPLETE_PERCENTAGE = ENV_TOKENS.get('COMPLETION_VIDEO_COMPLETE_PERCENTAGE',
```

Comment:
No need to add it here, it's already available https://github.com/edx/edx-platform/blob/859b6e91a067b7ec7ea23b4e53be0c3189d31900/cms/envs/production.py#L508-L511